src/scripts/galleryTabs.js

```diff
@@ -0,0 +1,288 @@
+  var picturesArea = slider.querySelector(".pictures-area");
+  var pos = 0;
+  var tempPos = 0;
+
+  var dataHolder = [];
+
+  function slideCreator() {
+    picturesArea.insertAdjacentHTML("afterbegin", '<div class ="slider-box slider-image"><img class="js-image js-image-ovl" src=""></div>');
+  }
+
+  function sliderIfnoHolder(k) {
+    var getProductImage = productBoxfinder[k].querySelector('.myImage').src;
+    var getProductName = productBoxfinder[k].querySelector('.content').innerText;
+    var getOldPrice = productBoxfinder[k].querySelector('.price-old').innerText;
+    var getNewPrice = productBoxfinder[k].querySelector('.price-new').innerText;
+    var getStars = productBoxfinder[k].querySelectorAll(".full").length;
+    var SliderVariables = {};
+    SliderVariables.src = getProductImage;
+    SliderVariables.name = getProductName;
+    SliderVariables.oldPrice = getOldPrice;
+    SliderVariables.newPrice = getNewPrice;
+    SliderVariables.stars = getStars;
+    dataHolder[dataHolder.length] = SliderVariables;
+  }
+
+  function topRatedChecker() {
+    var starAverage = 0;
+    for (var i = 0; i < productBoxfinder.length; i++) {
+      var starsHolder = productBoxfinder[i].querySelectorAll(".stars");
+      for (var j = 0; j < starsHolder.length; j++) {
+        var starsCounter = starsHolder[j].querySelectorAll(".full").length;
+        starAverage = starsCounter;
+      }
+      if (starAverage > 2) {
+        slideCreator();
+        sliderIfnoHolder(i);
+      }
+    }
+  }
+
+  function topSeller() {
+    var priceNew = 0;
+    var priceOld = 0;
+    for (var i = 0; i < productBoxfinder.length; i++) {
+      var getPriceNew = productBoxfinder[i].querySelectorAll('.price-new');
+      var getPriceOld = productBoxfinder[i].querySelectorAll('.price-old');
+      for (var j = 0; j < getPriceNew.length; j++) {
+        var getTextN = getPriceNew[j].innerText;
+        var remDolarN = getTextN.replace("$", "");
+        var priceNew = parseInt(remDolarN);
+      }
+      for (var k = 0; k < getPriceOld.length; k++) {
+        var getTextO = getPriceOld[k].innerText;
+        var remDolarO = getTextO.replace("$", "");
+        var priceOld = parseInt(remDolarO);
+      }
+      if ((priceOld / priceNew) > 1.4) {
+        slideCreator();
+        sliderIfnoHolder(i);
+      }
+    }
+  }
+
+  function topFeatures() {
+    var priceNew = 0;
+    var priceOld = 0;
+    var starAverage = 0;
+    for (var i = 0; i < productBoxfinder.length; i++) {
+      var getPriceNew = productBoxfinder[i].querySelectorAll('.price-new');
+      var getPriceOld = productBoxfinder[i].querySelectorAll('.price-old');
+      var starsHolder = productBoxfinder[i].querySelectorAll(".stars");
+      for (var j = 0; j < getPriceNew.length; j++) {
+        var getTextN = getPriceNew[j].innerText;
+        var remDolarN = getTextN.replace("$", "");
+        var priceNew = parseInt(remDolarN);
+      }

```

Comment:
na początku jest var priceNew = 0;
i tutaj znowu var priceNew = parse...
no i wygląda to tak, jakbyś za każdym razem nadpisywał tą zmienną, więc po co for?

Reply:
poniżej z priceOld podobnie